frontend/src/metabase-lib/parameters/utils/click-behavior.ts

```diff
@@ -132,11 +132,13 @@ function notRelativeDateOrRange({ type }: Parameter) {
 }
 
 export function getTargetsForQuestion(question: Question): Target[] {
-  if (question.isStructured()) {
+  const { isNative } = Lib.queryDisplayInfo(question.query());
+
+  if (!isNative) {
     return getTargetsForStructuredQuestion(question);
   }
 
-  if (question.isNative()) {
+  if (isNative) {
```

Comment:
We can get rid of the default `return []` now - this function should only have 1 `if` statement.

Reply:
I wonder why was `return []` there even before this change?
Because of the `InternalQuery`?

Reply:
Yeah, probably.